app/views/catarse_bootstrap/projects/_dashboard_user_settings.html.slim

```diff
@@ -1,124 +1,128 @@
-- unless disable_edit
-  .u-text-center
-    .w-container
-      .w-row
-        .w-col.w-col-2
-        .w-col.w-col-8
-          .fontweight-semibold.fontsize-large= t('.title')
-          p.fontsize-base= t('.subtitle_html')
-        .w-col.w-col-2.fontcolor-terciary
-
-= simple_form_for @project, defaults: {input_html: {disabled: disable_edit }, wrapper: :one_col, validation_text: false, label_wrapper_html: { class: 'w-col w-col-5 w-sub-col' } }, html: { id: 'project_account_form', class: 'w-form project-form' } do |f|
-  = hidden_field_tag :anchor, 'user_settings'
-  = f.simple_fields_for :account do |form|
-    .section
+#dashboard-user_settings-tab
+  - disable_edit = !policy(@project).update_account?
+  - unless disable_edit
+    .u-text-center
       .w-container
-        .w-col.w-col-8.u-marginbottom-30
-          = @project.display_errors(:user_settings)
-          - if disable_edit
-            .card.u-radius.card-message.u-marginbottom-20
-              .fontsize-small
-                = t('.account_edit_disabled_html', link: (link_to 'entrar em contato conosco'," #{CatarseSettings[:zendesk_url]}/account/dropboxes/#{CatarseSettings[:zendesk_dropbox_id]}", class: 'alt-link zendesk_widget' ))
-          .w-form.card.card-terciary
-            #email-form
-              .fontsize-base.fontweight-semibold= t('.user_data_title') 
-              .fontsize-small.u-marginbottom-20= t('.user_data_subtitle_html') 
+        .w-row
+          .w-col.w-col-2
+          .w-col.w-col-8
+            .fontweight-semibold.fontsize-large= t('.title')
+            p.fontsize-base= t('.subtitle_html')
+          .w-col.w-col-2.fontcolor-terciary
+
+  = simple_form_for @project, defaults: {input_html: {disabled: disable_edit }, wrapper: :one_col, validation_text: false, label_wrapper_html: { class: 'w-col w-col-5 w-sub-col' } }, html: { id: 'project_account_form', class: 'w-form project-form' } do |f|
+    = hidden_field_tag :anchor, 'user_settings'
+    = f.simple_fields_for :account do |form|
+      .section
+        .w-container
+          .w-col.w-col-8.u-marginbottom-30
+            = @project.display_errors(:user_settings)
+            - if disable_edit
+              .card.u-radius.card-message.u-marginbottom-20
+                .fontsize-small
+                  = t('.account_edit_disabled_html', link: (link_to 'entrar em contato conosco'," #{CatarseSettings[:zendesk_url]}/account/dropboxes/#{CatarseSettings[:zendesk_dropbox_id]}", class: 'alt-link zendesk_widget' ))
+            .w-form.card.card-terciary
+              #email-form
+                .fontsize-base.fontweight-semibold= t('.user_data_title') 
+                .fontsize-small.u-marginbottom-20= t('.user_data_subtitle_html') 
+                .w-row
+                  .w-col.w-col-6
+                    = form.input_field :person_type, as: :select, collection: ['Pessoa Física', 'Pessoa Jurídica'], class: 'w-select positive account_type', include_blank: false
```

Comment:
This does not need to be persisted.
I think this select field should just be used to call the changeType method in JS, but not go to the database.